.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- HTML Comments can be left in place or removed. -->
+
+#### Proposed Changes ####
+
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. -->
+
+<!-- Does this change require an update to documentation? -->
+
+#### Types of Changes ####
+
+<!-- What types of changes does your code introduce to RKE2 or k3s? Bugfix, New Feature, Breaking Change, etc -->
```

Comment:
We should either remove this section or modify it. Types of Changes could be like "issue validation", "patch validation", or "feature validation" I think

Reply:
done